airflow/www/templates/airflow/dags.html

```diff
@@ -41,14 +42,25 @@ <h2>DAGs</h2>
         </form>
       </div>
     </div>
+
+    <div id="owner-boxes">
+      {% for owner in dags|map(attribute='owners')|unique(case_sensitive=True)|sort %}
```

Comment:
can you put spaces in between:
```suggestion
      {% for owner in dags | map(attribute='owners') | unique(case_sensitive=True) | sort %}
```